### DISCUSSION TITLE
Multi-platform configuration - multiple instances?

### DISCUSSION THREAD

**Discussion by WilliamDEdwards** [renovatebot/renovate#23619] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

36.25.0

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I would like to run Renovate for both the GitLab and GitHub platforms.

The `platform` config key supports only a single value. This implies that multiple configs should be created, to periodically run an instance of `RENOVATE_CONFIG_FILE=... renovate` per platform.

However, certain resources (such as the `/tmp/renovate/` directory) would be shared between these instances. The documentation does not mention whether that is a problem or not.

### What you tried so far.

N/A

### Relevant debug logs

N/A

**Comment by rarkins** [comment, marked as answer]
Shared tmp is ok. Recommended to use env variables for any platform differences

**Reply by WilliamDEdwards** [reply]
What do you mean by 'recommended to use env variables'? Do you mean you'd prefer using environment variables rather than any config? Or omit `platform` and `token` from a single shared config and pass the environment variables `RENOVATE_TOKEN` and `RENOVATE_PLATFORM`? If so, why? Just to avoid duplication?

**Reply by rarkins** [reply]
Yes
